lms/static/scripts/frontend_apps/components/ContentSelector.tsx

```diff
@@ -333,9 +360,20 @@ export default function ContentSelector({
             onClick={() => selectDialog('canvasFile')}
             title="Select PDF from Canvas"
           >
-            Canvas
+            Canvas File
           </OptionButton>
         )}
+        {canvasPagesEnabled && (
+          <OptionButton
+            data-testid="canvas-page-button"
+            details="PAGE"
```

Comment:
```suggestion
            details="Page"
```

PDF is capitalized because it is an acronym. Other non-acronym terms in this UI position are not ("Web page", "Article").